README.md

```diff
@@ -80,7 +77,8 @@ class ChatroomApplication : Application() {
 
   override fun onCreate() {
     super.onCreate()
-
+    // ChatroomUIKitClient.getInstance().setUp() 可以在任意位置实现 
+    // 但需要保证在调用setUp()前尽量不去调用SDK相关API 防止未初始化SDK引起的NullPointerException异常
```

Comment:
// But you need to make sure not to call SDK related APIs before calling setUp() to prevent NullPointerException caused by uninitialized SDK